pkg/cmd/server/origin/auth.go

```diff
@@ -355,8 +357,11 @@ func (c *AuthConfig) getAuthenticationHandler(mux cmdutil.Mux, errorHandler hand
 				}
 				passwordSuccessHandler := handlers.AuthenticationSuccessHandlers{c.SessionAuth, redirectSuccessHandler{}}
 
+				loginPath := path.Join(OpenShiftLoginPrefix, identityProvider.Name)
```

Comment:
Only suffix /login if we're registering more than one login page, which is nicer